bluemira/materials/material.py

```diff
@@ -321,7 +323,7 @@ def from_dict(cls, name, materials_dict):
                 mat_dict[attr_name] = attr_type.deserialise(mat_dict[attr_name])
```

Comment:
Because `from __future__  import annotations` has been added all annotation types are now strings we got round this in `ParameterFrame` by using `get_type_hints` see https://github.com/Fusion-Power-Plant-Framework/bluemira/blob/aa4777a9462054768ab93de53bbc90b28444dce5/bluemira/base/parameter_frame/_frame.py#L69

Reply:
thanks